test/test_call.py

```diff
@@ -0,0 +1,109 @@
+from .support import HPyTest
+
+
+class TestCall(HPyTest):
+    def argument_combinations(self, *items):
```

Comment:
good idea! Maybe write it to take `**items` instead of `*items`? It would make the calls much nicer to read, and and I think that on modern Python the order of items in the dict is guaranteed, isn't it?

Reply:
I think it is. Let me try that.

Reply:
Woot. Implemented. Thank you for the suggestion.